obfuscation_test.go

```diff
@@ -0,0 +1,321 @@
+}{
+	{
+		sharedSecret: "b5756b9b542727dbafc6765a49488b023a725d631af688" +
+			"fc031217e90770c328",
+		ammagKey: "2f36bb8822e1f0d04c27b7d8bb7d7dd586e032a3218b8d414a" +
+			"fbba6f169a4d68",
+		stream: "e9c975b07c9a374ba64fd9be3aae955e917d34d1fa33f2e90f53" +
+			"bbf4394713c6a8c9b16ab5f12fd45edd73c1b0c8b33002df3768" +
+			"01ff58aaa94000bf8a86f92620f343baef38a580102395ae3abf" +
+			"9128d1047a0736ff9b83d456740ebbb4aeb3aa9737f18fb4afb4" +
+			"aa074fb26c4d702f42968888550a3bded8c05247e045b866baef" +
+			"0499f079fdaeef6538f31d44deafffdfd3afa2fb4ca9082b8f1c" +
+			"465371a9894dd8c2",
+		obfuscatedData: "69b1e5a3e05a7b5478e6529cd1749fdd8c66da6f6db4" +
+			"2078ff8497ac4e117e91a8cb9168b58f2fd45edd73c1b0c8b330" +
+			"02df376801ff58aaa94000bf8a86f92620f343baef38a5801023" +
+			"95ae3abf9128d1047a0736ff9b83d456740ebbb4aeb3aa9737f1" +
+			"8fb4afb4aa074fb26c4d702f42968888550a3bded8c05247e045" +
+			"b866baef0499f079fdaeef6538f31d44deafffdfd3afa2fb4ca9" +
+			"082b8f1c465371a9894dd8c2",
+	},
+	{
+		sharedSecret: "21e13c2d7cfe7e18836df50872466117a295783ab8aab0e" +
+			"7ecc8c725503ad02d",
+		ammagKey: "cd9ac0e09064f039fa43a31dea05f5fe5f6443d40a98be4071" +
+			"af4a9d704be5ad",
+		stream: "617ca1e4624bc3f04fece3aa5a2b615110f421ec62408d16c48e" +
+			"a6c1b7c33fe7084a2bd9d4652fc5068e5052bf6d0acae2176018" +
+			"a3d8c75f37842712913900263cff92f39f3c18aa1f4b20a93e70" +
+			"fc429af7b2b1967ca81a761d40582daf0eb49cef66e3d6fbca02" +
+			"18d3022d32e994b41c884a27c28685ef1eb14603ea80a204b2f2" +
+			"f474b6ad5e71c6389843e3611ebeafc62390b717ca53b3670a33" +
+			"c517ef28a659c251",
+		obfuscatedData: "08cd44478211b8a4370ab1368b5ffe8c9c92fb830ff4" +
+			"ad6e3b0a316df9d24176a081bab161ea0011585323930fa5b9fa" +
+			"e0c85770a2279ff59ec427ad1bbff9001c0cd1497004bd2a0f68" +
+			"b50704cf6d6a4bf3c8b6a0833399a24b3456961ba00736785112" +
+			"594f65b6b2d44d9f5ea4e49b5e1ec2af978cbe31c67114440ac5" +
+			"1a62081df0ed46d4a3df295da0b0fe25c0115019f03f15ec86fa" +
+			"bb4c852f83449e812f141a93",
+	},
+
+	{
+		sharedSecret: "3a6b412548762f0dbccce5c7ae7bb8147d1caf9b5471c3" +
+			"4120b30bc9c04891cc",
+		ammagKey: "1bf08df8628d452141d56adfd1b25c1530d7921c23cecfc749" +
+			"ac03a9b694b0d3",
+		stream: "6149f48b5a7e8f3d6f5d870b7a698e204cf64452aab4484ff1de" +
+			"e671fe63fd4b5f1b78ee2047dfa61e3d576b149bedaf83058f85" +
+			"f06a3172a3223ad6c4732d96b32955da7d2feb4140e58d86fc0f" +
+			"2eb5d9d1878e6f8a7f65ab9212030e8e915573ebbd7f35e1a430" +
+			"890be7e67c3fb4bbf2def662fa625421e7b411c29ebe81ec67b7" +
+			"7355596b05cc155755664e59c16e21410aabe53e80404a615f44" +
+			"ebb31b365ca77a6e",
+		obfuscatedData: "6984b0ccd86f37995857363df13670acd064bfd1a540" +
+			"e521cad4d71c07b1bc3dff9ac25f41addfb7466e74f81b3e5455" +
+			"63cdd8f5524dae873de61d7bdfccd496af2584930d2b566b4f8d" +
+			"3881f8c043df92224f38cf094cfc09d92655989531524593ec6d" +
+			"6caec1863bdfaa79229b5020acc034cd6deeea1021c50586947b" +
+			"9b8e6faa83b81fbfa6133c0af5d6b07c017f7158fa94f0d206ba" +
+			"f12dda6b68f785b773b360fd",
+	},
+
```

Comment:
Please remove the extra new lines in-between successive elements of the slice literal.